doc/importexport.rst

```diff
@@ -76,8 +76,114 @@ There are no parameters for this method other than the file path to import.
+
+Setting Extra Options
+----------------------
+
+There are multiple options that can be set when exporting an object to a STEP file, but those can only be accessed by calling
+the :py:meth:`Shape.exportSTEP`` method on the lower level CadQuery object. For an explanation of the options available, see
+the documentation for that method.
+
+.. code-block:: python
+    # Create a simple object
+    box = cq.Workplane().box(10, 10, 10)
+
+    # Export the box
+    box.val().exportStep("/path/to/step/box.step", write_pcurves=True, precision_mode=1)
+
+Exporting Assemblies to STEP
+#############################
+
+It is possible to export CadQuery assemblies directly to STEP. The STEP exporter has multiple options which change the way
+exported STEP files will appear and operate when opened in other CAD programs. All assembly export methods shown here will
+preserve the color information from the assembly.
+
+Default
+--------
+
+CadQuery assemblies have a :py:meth:`Assembly.save` method which can write an assembly to a STEP file. An example assembly
+export with all defaults is shown below.
+
+.. code-block:: python
+    import cadquery as cq
+
+    # Create a sample assembly
+    assy = cq.Assembly()
+    body = cq.Workplane().box(10, 10, 10)
+    assy.add(body, color=cq.Color(1, 0, 0), name="body")
+    pin = cq.Workplane().center(2, 2).cylinder(radius=2, height=20)
+    assy.add(pin, color=cq.Color(0, 1, 0), name="pin")
+
+    # Save the assembly to STEP
+    assy.save('/path/to/step/output/directory', exporters.ExportTypes.STEP)
+
+This will produce a STEP file that is nested with auto-generated object names. The colors of each assembly object will be
+preserved, but the names that were set for each will not.
+
+Fused
+------
+
+The following will attempt to create a single, fused shape while preserving the name and color information of each assembly
+object. The process of fusing the solid may cause performance issues in some cases, and is likely to alter the faces of the
+fused solids.
+
+.. code-block:: python
+    import cadquery as cq
+
+    # Create a sample assembly
+    assy = cq.Assembly()
+    body = cq.Workplane().box(10, 10, 10)
+    assy.add(body, color=cq.Color(1, 0, 0), name="body")
+    pin = cq.Workplane().center(2, 2).cylinder(radius=2, height=20)
+    assy.add(pin, color=cq.Color(0, 1, 0), name="pin")
+
+    # Save the assembly to STEP
+    assy.save('/path/to/step/output/directory', exporters.ExportTypes.STEP, export_mode=exporters.STEPExportMode.FUSED)
+
+Naming
+-------
+
+It is also possible to set the name of the top level assembly object in the STEP file with either the DEFAULT or FUSED methods.
+This is done by setting the name property of the assembly before calling :py:meth:`Assembly.save`.
+
+.. code-block:: python
+    assy = Assembly(name="my_assembly")
+    assy.save('/path/to/step/output/directory', exporters.ExportTypes.STEP, export_mode=exporters.STEPExportMode.FUSED)
```

Comment:
directory issue again.  

Also I notice none of the examples use literals.  Perhaps convert an examlpe to show use of literals.

assy.save("out.step", exportMode="fused")
and
assy.save("out.stp", "STEP")